apps/address-service/domains/address/schema/LinkAddressAndSourceService.js

```diff
@@ -0,0 +1,124 @@
+/**
+ * Generated by `createservice address.LinkAddressAndSourceService --type mutations`
+ */
+
+const { parseAddressesFromString } = require('@open-condo/clients/address-service-client/utils')
+const { GQLError, GQLErrorCode: { BAD_USER_INPUT } } = require('@open-condo/keystone/errors')
+const { GQLCustomSchema, find } = require('@open-condo/keystone/schema')
+
+const access = require('@address-service/domains/address/access/LinkAddressAndSourceService')
+const { EMPTY_SOURCE_ERROR, EMPTY_ADDRESS_ERROR, SOURCE_ALREADY_EXISTS_ERROR, INCORRECT_ADDRESS_SOURCE_ERROR, INCORRECT_ADDRESS_ERROR } = require('@address-service/domains/address/constants')
+const { AddressSource } = require('@address-service/domains/address/utils/serverSchema')
+const { mergeAddressAndHelpers } = require('@address-service/domains/common/utils/services/search/searchServiceUtils')
+
+const ERRORS = {
+    EMPTY_SOURCE: {
+        query: 'linkAddressAndSource',
+        code: BAD_USER_INPUT,
+        type: EMPTY_SOURCE_ERROR,
+        message: 'Source is empty',
+    },
+    EMPTY_ADDRESS: {
+        query: 'linkAddressAndSource',
+        code: BAD_USER_INPUT,
+        type: EMPTY_ADDRESS_ERROR,
+        message: 'Address is empty',
+    },
+    INCORRECT_ADDRESS_SOURCE: {
+        query: 'linkAddressAndSource',
+        code: BAD_USER_INPUT,
+        type: INCORRECT_ADDRESS_SOURCE_ERROR,
+        message: 'Incorrect address source',
+    },
+    INCORRECT_ADDRESS: {
+        query: 'linkAddressAndSource',
+        code: BAD_USER_INPUT,
+        type: INCORRECT_ADDRESS_ERROR,
+        message: 'Incorrect address',
+    },
+    SAME_SOURCE: {
+        mutation: 'linkAddressAndSource',
+        code: BAD_USER_INPUT,
+        type: SOURCE_ALREADY_EXISTS_ERROR,
+        message: 'This source already exists',
+    },
+}
+
+const LinkAddressAndSourceService = new GQLCustomSchema('LinkAddressAndSourceService', {
+    types: [
+        {
+            access: true,
+            type: 'input LinkAddressAndSourceInput { dv: Int!, sender: SenderFieldInput!, source: String!, tin: String, address: AddressWhereUniqueInput!, parseUnit: Boolean }',
```

Comment:
Maybe it's better to add separated input object for helpers? It will make functions signature more consistent with address service api.
```suggestion
            type: 'input LinkAddressAndSourceInput { dv: Int!, sender: SenderFieldInput!, source: String!, helpers: LinkAddressAndSourceHelpersInput, address: AddressWhereUniqueInput!, parseUnit: Boolean }',
```
Or use existing input type for helpers